
<launch>

  <node name="dji_gimbal_camera_action_server" pkg="sensyn_dji_gimbal_action" type="dji_gimbal_camera_action_server" output="screen">
  </node>

  <node name="test_gimbal_client" pkg="sensyn_dji_gimbal_action" type="test_gimbal_client" output="screen">
  </node>

  <node name="dummy_gimbal_node" type="dummy_gimbal_node.py" pkg="sensyn_dji_gimbal_action" output="screen"/>


</launch>